### PR TITLE
Updating rbts-takehome-device-setup.md (#Issue1561)

### PR DESCRIPTION
**Pull request by Jackbui96**
Gitraw preview: https://rawgit.com/Jackbui96/Jackbui96.github.io/issue1561/#!pages/robots/rbts-takehome-device-setup.md

### CODE REVIEW

**Comment by xyb994** [comment]
@Jackbui96 Thank you for adding screenshots and instructions to the Android debugging setup guide

- As long as we accurately document all the keywords to enable USB debugging, the process is relatively straight forward. A nice example is the Android Studio official user guide at <https://developer.android.com/studio/debug/dev-options.html#enable>. In other words, less is more. In this case, is "Before Activating "Developer Options"" really necessary?
- Do you think some circle/highlight on the screenshot might help our readers? 
- Since external images might not last forever, could you upload all screenshots to `/pages/robots/images/`?
- Could you fix the grammar error at [line 10](https://github.com/open-learning-exchange/open-learning-exchange.github.io/pull/1615/files#diff-8b0685030861a33fbc3535b32fd9a7f2R10)? "Here’s we will show you how to activate "Developer Options"."

**Comment by Jackbui96** [comment]
@xyb994 After reading through the guide again, I agree with you that some of the informations are redundant, and it is best not to add them into the guide. Plus if we can keep the guide short, it will be easier for the user to follow.

For the screen shot, I will try to circle/highlight the keywords that is important to the user.

About the image files, I didnt notice that we have a place where we would store all of our resources. Thank you for pointing that out for me.

I will also fix the grammar error at line 10. Again, thank you for letting me know.

I will update the file shortly.

**Comment by xyb994** [comment]
@Jackbui96 Could you remove`rbts-device-transfer-file.jpg` since we are not embedding it in the Markdown.

**Comment by Jackbui96** [comment]
Ok sure, I will remove it right away

**Comment by Jackbui96** [comment]
@xyb994 is there anything else that need to be done? Or this issue is ready to be merge?

File: pages/robots/rbts-takehome-device-setup.md [hunk 1]
```diff
@@ -1,43 +1,65 @@
...
+Follow these directories to find 'Build number' on a few popular devices.
 
-LG G3: Settings > About phone > Software information > Build number
+    Stock Android: Settings > About phone > Build number
+    Samsung Galaxy S5: Settings > About device > Build number
+    LG G3: Settings > About phone > Software information > Build number
+    HTC One (M8): Settings > About > Software information > More > Build number
 
-HTC One (M8): Settings > About > Software information > More > Build number
+![BuildNumber](images/rbts-device-build-number.jpg)
```
Comment (xyb994) [inline pages/robots/rbts-takehome-device-setup.md]: @Jackbui96 Can you also add space between words? "BuildNumber" will become `alt="BuildNumber"` in HTML and this is not so good in terms of accessibility. See more at https://ux.shopify.com/writing-for-all-people-how-to-use-alternative-text-well-1205a18307a1

Comment (Jackbui96) [reply pages/robots/rbts-takehome-device-setup.md]: @xyb994 Thank you for pointing that out to me. I did not know the importance of Alternative Text before reading the article that you sent me, now I know why spacing and clarification is important. I also finished updating the Alternative Text, please review.

**Pull request merged**